openedx/core/djangoapps/content/course_overviews/models.py

```diff
@@ -101,9 +101,9 @@ class Meta(object):
     eligible_for_financial_aid = BooleanField(default=True)
 
     @classmethod
-    def _create_from_course(cls, course):
+    def _create_or_update_from_course(cls, course):
```

Comment:
Consider renaming this to simply `_create_or_update`.  See other related comment.